src/bin/jsii-release-golang.ts

```diff
@@ -0,0 +1,12 @@
+import * as go from '../go';
```

Comment:
Don't we need a shebang here?

Reply:
No required since its not actually invoked as a script, but yeah i'll add it so it can be.